Doc/library/functions.rst

```diff
@@ -1262,7 +1262,8 @@ are always available.  They are listed here in alphabetical order.
    closest multiple of 10 to the power minus *ndigits*; if two multiples are
```

Comment:
while we are modifying the doc, I dislike the start: "Return the floating point value *number*": number can be an integer. round(123, -2) doesn't use floating point numbers. Maybe rephrase to following text?

Return  *number* rounded to (...). *number* can be an integer or a floating point number.

Or just remove "floating pointer number"?